doc/api/diagnostics_channel.md

```diff
@@ -307,6 +307,108 @@ channel.subscribe(onMessage);
 channel.unsubscribe(onMessage);
 ```
 
+## Example
+
+```mjs
+import http from 'node:http';
+import dc from 'node:diagnostics_channel';
+
+const onClientRequestStart = dc.channel('http.client.request.start');
+const onClientResponseFinish = dc.channel('http.client.response.finish');
+const onServerRequestStart = dc.channel('http.server.request.start');
+const onServerResponseFinish = dc.channel('http.server.response.finish');
```

Comment:
Rather than all these specific channels making one very large and use-case specific example, perhaps it would be better to have a more generic example and instead have a list of currently provided internal channels somewhere in the doc so each channel can have its messages documented more clearly?

Reply:
> Rather than all these specific channels making one very large and use-case specific example, perhaps it would be better to have a more generic example and instead have a list of currently provided internal channels somewhere in the doc so each channel can have its messages documented more clearly?

Thanks for review ! I have updated the docs, please review again !